package.json

```diff
@@ -31,7 +31,7 @@
         "clean": "rimraf dist && rimraf dist_browser",
         "build": "genversion --semi --es6 src/PackageVersion.js && tsc && cross-env babel src --out-dir dist && cross-env NODE_ENV=production webpack -p --config webpack.config.js",
         "docs": "jsdoc -c jsdoc.json",
-        "prepublishOnly": "npm run types-lint && npm run clean && npm run build:browser && npm run docs",
+        "prepublishOnly": "npm run types-lint && npm run clean && npm run build && npm run docs",
         "types-lint": "tsc && dtslint --localTs node_modules/typescript/lib types"
```

Comment:
Linting seems to pick up on an issue when running `npm run types-lint`:
 
`ERROR: 111:17  interface-over-type-literal  Use an interface instead of a type literal.`

See https://palantir.github.io/tslint/rules/interface-over-type-literal/ about what this rule effects.

We either can either set this rule to `false` in the `types/tslint.json` file or prefer interfaces over types.